service-block/src/main/java/blocks/service/AbstractBlock.java

```diff
@@ -13,6 +13,11 @@
         });
     }
 
+    @Override
+    public BlockRef<T> ref() {
+        return getRef(this.getClass().getName());
```

Comment:
This will make the reference names in health etc a bit less nicely controlled but there's always a way to keep it the old way by submitting reference yourself, so fine with this 👍 Will make it cleaner if you don't care about this :)

Reply:
Agreed, however the name in health will still be very clear (unless you mess up your class name... then I can't help ;-)). But also saves unnecessary code when constructing service, which for me is a bigger benefit.